Parse-Dashboard/app.js

```diff
@@ -48,11 +48,16 @@ function checkIfIconsExistForApps(apps, iconsFolder) {
   }
 }
 
-module.exports = function(config, allowInsecureHTTP) {
+module.exports = function(config, allowInsecureHTTP, trustProxy) {
```

Comment:
Instead of adding a new parameter here, can you make this part of the `config` parameter?